composer.json

```diff
@@ -12,13 +12,18 @@
     "require": {
         "php": "^7.3",
         "nexylan/slack": "^3.0",
+        "php-http/discovery": "^1.6",
```

Comment:
Needed because we depend on Psr17FactoryDiscovery, which was introduced in 1.6 of discovery. We didn't catch this before because there was no integration test and so the `--prefer-lowest` didn't hit this.